### PR TITLE
[WIP] 0.11.0 release notes

### PR DESCRIPTION
**Pull request by mzmssg**
Please help review and add more items.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Release v0.11.0 #
+
+## New Features ##
+
+* Support team wise NFS storage, including:
+    - A NFS configuration plug-in and a commandline tool. [#2346](https://github.com/Microsoft/pai/pull/2346)
```
Comment (fanyangCS) [inline RELEASE_NOTES.md]: An NFS ...

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* Filter supports in Job List View. [#302](https://github.com/Microsoft/pai/pull/302)
+    ![](./docs/release_note/images/filter.png)
+
+* Hold the Env for failed jobs which are casued by user error. [#2272](https://github.com/Microsoft/pai/pull/2272)
+
+
+## Improvements ##
+
+### Service ###
```
Comment (abuccts) [inline RELEASE_NOTES.md]: Is `## ... ##` or `### ... ###` necessary?
`##` or `###` is enough for [markdown headers](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#headings).

Comment (mzmssg) [reply RELEASE_NOTES.md]: Sure

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    ![](./docs/release_note/images/filter.png)
+
+* Hold the Env for failed jobs which are casued by user error. [#2272](https://github.com/Microsoft/pai/pull/2272)
+
+
+## Improvements 
+
+### Service 
+
+* Webportal:
```
Comment (abuccts) [inline RELEASE_NOTES.md]: There are many trailing spaces in this file, please also remove them.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# Release v0.11.0
+
+## New Features
+
+* Support team wise NFS storage, including:
+    - An NFS configuration plug-in and a commandline tool. [#2346](https://github.com/Microsoft/pai/pull/2346)
+    - A simple NFS-job submit plug-in. [#2358](https://github.com/Microsoft/pai/pull/2358)
+
```
Comment (scarlett2018) [inline RELEASE_NOTES.md]: Refer to [Simplified Job Submission for OpenPAI + NFS deployment](https://github.com/Microsoft/pai/wiki/Simplified-Job-Submission-for-OpenPAI-with-NFS-deployment) for more details.

**Pull request merged**